crates/noirc_evaluator/src/ssa_refactor/ssa_gen/context.rs

```diff
@@ -322,7 +325,7 @@ impl<'a> FunctionContext<'a> {
         // pre-loop
         let result_alloc = self.builder.insert_allocate();
         let true_value = self.builder.numeric_constant(1u128, Type::bool());
-        self.builder.insert_store(result_alloc, true_value);
+        self.builder.set_location(location).insert_store(result_alloc, true_value);
```

Comment:
We should probably set the location on line 326 where the first instruction is inserted

Reply:
Done